### PR TITLE
docs: improve browser example code and add list demo

### PR DESCRIPTION
**Pull request by olizilla**
it look fun now! **and** it demo `put` and `list`!

Screenshot 2021-07-28 at 21 58 23

depends on #198

License: (Apache-2.0 AND MIT)
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Review by vasco-santos — APPROVED** [review]
LGTM ❤️

File: packages/client/examples/browser/main.js [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
+  showMessage(`> ✅ web3.storage now hosting ${cid}`)
+  showLink(`https://dweb.link/ipfs/${cid}`)
 
-  const text = document.createTextNode(`Content added with CID: ${cid}`)
-  cidP.appendChild(text)
+  showMessage('> 📡 fetching the list of all unique uploads on this account')
+  let totalBytes = 0
+  for await (const upload of client.list()) {
+    showMessage(`> 📄 ${upload.cid}  ${upload.name}`)
+    totalBytes += upload.dagSize
```
Comment (alanshaw) [inline packages/client/examples/browser/main.js:32]:
```suggestion
    totalBytes += upload.dagSize || 0
```

File: packages/client/examples/browser/package.json [hunk 2]
```diff
@@ -9,6 +10,6 @@
     "vite": "^2.3.7"
   },
   "dependencies": {
-    "web3.storage": "^1.0.0"
+    "web3.storage": "file:../.."
```
Comment (alanshaw) [inline packages/client/examples/browser/package.json:13]: When published can it please depend on a real version so users don't have to npm install in the root as well as here?

File: packages/client/examples/browser/main.js (hunk 1 above)
Comment (olizilla) [reply packages/client/examples/browser/main.js:32]: not need, as dagSize is either a number or `null`
```console
> x = 0
0
> x += null
0
```

but
```console
> x += undefined
NaN
```
so will do for best safety

File: packages/client/examples/browser/package.json (hunk 2 above)
Comment (olizilla) [reply packages/client/examples/browser/package.json:13]: yes, this was just for testing until the latest module release, which now!

**Comment by yusefnapora** [comment]
This is awesome! Nice work @olizilla

**Comment by olizilla** [comment]
the thing to store things is now stored on the thing!
https://bafybeigxve4jcuc5o5cq3sjifgw5ukzteb23xsqalzjltavk5bzn4bi3w4.ipfs.dweb.link/

**Review by alanshaw — APPROVED** [review]

**Pull request merged**